app/models/user.rb

```diff
@@ -1,5 +1,27 @@
 class User < ApplicationRecord
+  include UsersHelper
```

Comment:
Those are **view** helpers and they are not supposed to be included into the model.
If you need `age` method in the model it means that it's not used in the view only - you can move it from the helper to the model, and then remove it from the helper, i.e. rewrite `UsersHelpers#user_age` to `User#age`.